### PR TITLE
update header and add page component

### LINKED ISSUE #75: Update header to match design
```[tasklist]
### Tasks
- [ ] #77 
```

### PR DESCRIPTION
**Pull request by DasProffi**
closes #75

### CODE REVIEW

**Comment by DasProffi** [comment]
Either create a new issue for adding the breadcrumbs to the header or do this in this PR after #176 is closed

File: tasks/pages/contact.tsx [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
       <Header
         title="helpwave"
-        navigation={[
-          { text: 'Dashboard', href: '/' },
-          { text: 'Contact', href: '/contact' }
-        ]}
-        actions={[
-          <div key="0" className={tw('w-40')} />
-        ]}
       />
```
Comment (use-to) [inline tasks/pages/contact.tsx:10]: Thats duplicated in `/contact`

File: tasks/pages/organizations/[uuid].tsx [hunk 2]
```diff
@@ -72,19 +70,21 @@ const WardsPage: NextPage = ({ language }: PropsWithLanguage<WardsPageTranslatio
...
+    <PageWithHeader
+      leftSide={[(
+        <BreadCrumb
+          key="crumbs"
+          crumbs={[
+            { display: translation.organizations, link: '/organizations' },
+            { display: translation.wards, link: `/organizations/${organizationUUID}` }
+          ]}
+        />
+      )]}
```
Comment (use-to) [inline tasks/pages/organizations/[uuid].tsx:82]: Maybe we should try to implement a way to skip the usage of the `BreadCrumb` component on the actual pages/views.

One simple idea:
1. Create an additional attribute/parameter passing the current navigation stack
2. Prepend the `BreadCrumb`-component to the internal/private `leftSide` array.

Comment (use-to) [reply tasks/pages/organizations/[uuid].tsx:82]: This would increase the developer experience (such as readability)!

**Review by use-to — CHANGES_REQUESTED** [review]
Created new issue #183

**Review by use-to — APPROVED** [review]
Alright. No further questions your honor...

**Pull request merged**